scripts/configure_conda

```diff
@@ -96,7 +96,7 @@ function activate_environment() {
     then
         echo conda env update --file "$file"
     else
-        conda env update --file "$file" 2>/tmp/${0##*/}_${logfile}.log
+	conda xxxxx env update --file "$file" 2> >(tee /tmp/${0##*/}_${logfile}.log)
```

Comment:
I see some xxxx in the conda command, is that intended?

Reply:
Thank you Francesco for reading more carefully than me :-)

No I just forgot to remove the test setup raising a conda error :-( ... fixed.